ethdb/README.md

```diff
@@ -0,0 +1,51 @@
+## ethdb
+
+Parity Ethereum interaction with the DB on-demand.
+
+### Usage
+
+```
+Parity Ethereum interaction with the DB on-demand.
```

Comment:
That will probably get quickly outdated. If we stick with docopt perhaps we could import that content from the same file used in the binary?

Reply:
i've addressed this in commit ee203a618471d34514140fa6188ddbf147d3635c